starter/script.js

```diff
@@ -1 +1,69 @@
-// put code here!
+var name = prompt ("What's your name"); alert ("Nice to meet you" + " " + name + "." + " " + "You just won a free vacation!");
+
+var destination = prompt ("Pick a destination: Fiji, Italy, South Africa (F,I,S) ");
+
+if (destination==="F") {
+  alert ("Great! Fiji has excellent beaches.");
+  var whoIsGoing = prompt ("How many people, including you, will be going?");
+
+        if (whoIsGoing < 3) {
+            alert ("Fabulous! We’ll book you and your guests at the Fiji Cabana located right on the beach.");
+        }
+        else if (whoIsGoing < 5 ) {
+            alert ("You’ll need a little more room so we’ll book a two-bedroom suite for you and your guests.");
+        }
+        else {
+            alert ("Woah! That’s more people than we expected. You’ll have to foot some of the bill!");
+        }
+
+  } else if (destination==="I") {
+    alert ("Good choice! Your going to love Italy!");
+    var whoIsGoing = prompt ("How many people, including you, will be going?");
+
+        if (whoIsGoing < 3) {
+            alert ("Fabulous! We’ll book you and your guests on the riviera.");
+        }
+        else if (whoIsGoing < 5 ) {
+            alert ("Great! There's an awesome hotel a few blocks from the Leaning Tower of Pisa.");
+        }
+        else {
+            alert ("Woah! That’s more people than we expected. You’ll have to foot some of the bill!");
+        }
+  }
+  else if (destination==="S") {
```

Comment:
Take a look at [switch...case](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/switch). It saves you a lot of trouble when you're writing something like:

```
if x === "a"
else if x === "b"
else if x === "c"
else if x === "d"
...
```